instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/patch.py

```diff
@@ -111,12 +138,88 @@ async def traced_method(wrapped, instance, args, kwargs):
                 return result
 
             except Exception as error:
+                error_type = type(error).__name__
```

Comment:
same here, it should be 
```suggestion
                error_type = type(error).__qualname__
```

as implemented inside `handle_span_exception`